<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetSetting</class>
 <widget class="QWidget" name="NetSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_NetSet">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>网络设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_LocalIP">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>本机IP：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_ServerIP">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>服务器IP：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Protocol">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>协议：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Prot">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_LocalIP">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_ServerIP">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Port">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>220</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_Protocol">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>UDP</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pb_Set">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>400</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Exit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>400</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_Work_Mode">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>实验室模式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>在线模式</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lb_work_mode">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>工作模式</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
